db/db_impl/db_impl.cc

```diff
@@ -1956,19 +1961,38 @@ std::vector<Status> DBImpl::MultiGet(
   StopWatch sw(immutable_db_options_.clock, stats_, DB_MULTIGET);
   PERF_TIMER_GUARD(get_snapshot_time);
 
-#ifndef NDEBUG
-  for (const auto* cfh : column_family) {
-    assert(cfh);
-    const Comparator* const ucmp = cfh->GetComparator();
+  size_t num_keys = keys.size();
+  std::vector<Status> stat_list(num_keys);
+
+  bool should_fail = false;
+  for (size_t i = 0; i < num_keys; ++i) {
+    assert(column_family[i]);
```

Comment:
We can assert the equality of `sizes()` for `column_family` and `keys`, but not for `values` and `timestamps` because we later do `resize()` for the latter two.